bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/SeqConfigIgrins2CB.java

```diff
@@ -0,0 +1,30 @@
+package edu.gemini.spModel.gemini.igrins2;
+
+import edu.gemini.pot.sp.ISPSeqComponent;
+import edu.gemini.spModel.config.HelperSeqCompCB;
+import edu.gemini.spModel.data.config.IConfig;
+
+import java.util.Map;
+
+final public class SeqConfigIgrins2CB extends HelperSeqCompCB {
```

Comment:
I'm not totally sure this is needed, it was essentially copied from the Ghost model